pkg/kubernetes/views/registry-dashboard-page.html

```diff
@@ -111,10 +113,51 @@ <h2 class="card-pf-title">
                 <span translatable="yes">Your login credentials do not give you access to use the docker registry from the command line.</span>
             </div>
 <code ng-if="settings.registry.password">$ sudo docker login -p {{settings.registry.password}} -e unused -u unused <span ng-class="{ placeholder: !settings.registry.host_explicit }">{{settings.registry.host}}</span></code>
-            <p translatable="yes">Push an image:</p>
+            <p ng-if="settings.registry.openshifthost_explicit">
+                <span translatable="yes"></span>
+                Log into OpenShift:
```

Comment:
> The wording here gives me many questions.

I agree. This is a small attempt to connect the registry interface with the OpenShift world. Yes, it assumes some knowledge. We could add some language below the command, even a link to the OpenShift web console, but I'm not sure that is all that helpful.